.github/workflows/api-cle-only.yml

```
@@ -0,0 +1,72 @@
+env:
+  SUBFOLDER_NAME: api
+  APP_NAME: snu-api-cle
```

Tu as mis `snu-cle-admin` et `snu-cle-app` donc pour plus de cohérence je mettrais

```suggestion
  APP_NAME: snu-cle-api
```